src/main/java/ladder/controller/LadderController.java

```diff
@@ -2,27 +2,27 @@
 
 import ladder.controller.dto.LadderRequest;
 import ladder.domain.*;
-import ladder.factory.RowFactory;
+import ladder.factory.RandomRowStrategy;
+import ladder.factory.RowStrategy;
 import ladder.view.InputView;
 import ladder.view.ResultView;
 
 public class LadderController {
 
     private final InputView inputView;
     private final ResultView resultView;
-    private final RowFactory rowFactory;
 
-    public LadderController(InputView inputView, ResultView resultView, RowFactory rowFactory) {
+    public LadderController(InputView inputView, ResultView resultView) {
         this.inputView = inputView;
         this.resultView = resultView;
-        this.rowFactory = rowFactory;
     }
 
     public void run() {
         LadderRequest request = new LadderRequest(inputView.names(), inputView.results(), inputView.ladderHeight());
```

Comment:
```suggestion
        LadderRequest request = new LadderRequest(InputView.names(), InputView.results(), InputView.ladderHeight());
```

저는 내부 상태를 가지고 있지 않으므로, 그냥 메소드로 static으로 만들고 호출할것 같네요 🤔